<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectInstrument</class>
 <widget class="QDialog" name="SelectInstrument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Instrument</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Current instrument:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentInstrument">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="instrumentGenreFilter"/>
   </item>
   <item>
    <widget class="Ms::FilterableTreeWidget" name="instrumentList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>0</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Ms::SearchBox" name="instrumentSearch">
     <property name="placeholderText">
      <string>Search</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ms::FilterableTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widgets/filterabletreeview.h</header>
  </customwidget>
  <customwidget>
   <class>Ms::SearchBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/searchbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>currentInstrument</tabstop>
  <tabstop>instrumentGenreFilter</tabstop>
  <tabstop>instrumentList</tabstop>
  <tabstop>instrumentSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectInstrument</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectInstrument</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
